src/Sylius/Component/Attribute/Model/AttributeValue.php

```diff
@@ -110,7 +134,10 @@ public function getValue()
      */
     public function setValue($value)
     {
-        $this->value = $value;
+        $this->assertAttributeIsSet();
+
+        $setter = 'set'.ucfirst($this->attribute->getStorageType());
```

Comment:
Which class name? I'd say it is a rule that storageType results in corresponding property and getter/setter.